wagtail/admin/templates/wagtailadmin/workflows/edit.html

```diff
@@ -59,7 +58,14 @@
                                 <ul>
                                     {% if can_enable %}
                                         <li>
-                                            <button type="submit" class="button no" data-enable-action>{{ view.enable_item_label }}</button>
+                                            <button class="button no" 
+                                                type="submit"
+                                                data-controller="w-action"
+                                                data-w-action-csrf-token-value = '{{ csrf_token|escapejs }}'
+                                                data-w-action-key-name-value = 'csrfmiddlewaretoken'
+                                                data-w-action-post-url-value = '{{ view.get_enable_url }}'
+                                                data-action="w-action#enableAction">{{ view.enable_item_label }}
```

Comment:
I recommend we try something a bit more generic - this action may not always be enabling something. 

Some ideas
- `action` (could be confusing)
- `trigger` 
- `go`
- `submit` - this could work - it's on the submit button 
- `post` - this feels like it might be best as it's a bit more explicit that we're doing a POST request 

Write each one out in your html locally and see what you think feels best. One word - if possible - is best. 


```suggestion
                                                data-action="w-action#post">{{ view.enable_item_label }}
```